index.php

```diff
@@ -38,12 +38,42 @@
 		</div>
 		<?php endif; ?>
 
+
+
+
  		<div class="pull-right visible-desktop home-lab">
 			<a data-toggle="modal" href="#tutorial_video_modal"><img src="img/screencast_thumb.png"></a>
 		</div>
-		<h1><?php echo $homepage_title; ?> <small><?php echo $homepage_subtitle; ?></small></h1>
 
-		<p class="lead">You can use labrador to find and download processed data or request new datasets.
+<h1><img class="pull-left visible-desktop" src=<?php echo $logo; ?> style="margin: 0 20px 10px 0;">
+
+		<?php echo $homepage_title; ?> <small><?php echo $homepage_subtitle; ?></small></h1>
+
+		<p class="lead">
+
+                <?php 
+
+                if(isset($data_root_path)){
+
+                echo 'System Status: &nbsp;';
+
+                $data_root_path_no_slash  = str_replace('/', '', $data_root_path);
```

Comment:
What's this line for? What happens if someone sets their root path to a subdirectory? (eg. `/path/to/data`)